### PR TITLE
(SIMP-3196) Remove auth.conf from simp-adapter

### PR DESCRIPTION
**Pull request by jeefberkey**
The puppetserver's auth.conf file is now managed with puppet.

SIMP-3196 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
Related PR in pupmod https://github.com/simp/pupmod-simp-pupmod/pull/57

File: src/assets/simp-adapter/build/simp-adapter.spec [hunk 1]
```diff
@@ -214,7 +211,7 @@ puppet config set stringify_facts false || :
 (
   cd %{puppet_confdir}
 
-  simp_overrides='hiera.yaml auth.conf'
+  simp_overrides='hiera.yaml'
```
Comment (lnemsick-simp) [inline src/assets/simp-adapter/build/simp-adapter.spec]: Was this the only mechanism to back up an existing auth.conf file?  I'm concerned that we will silently wipe out an existing file with the use of the Puppet auth.conf resource, upon upgrade.  If we can back it up, that would be great. If that is not feasible, this should be in the CHANGELOG.

Comment (nick-markowski) [inline src/assets/simp-adapter/build/simp-adapter.spec]: @lnemsick-simp I will address this with https://simp-project.atlassian.net/browse/SIMP-3285, in pupmod-simp-pupmod

**Comment by nick-markowski** [comment]
FYI: to test, pull in this and SIMP-3283

**Pull request merged**